Resources/Prototypes/Entities/Clothing/Hands/armwarmers.yml

```diff
@@ -0,0 +1,53 @@
+- type: entity
+  parent: ClothingHandsGlovesSyntheticBase
+  id: ClothingHandsArmWarmers
+  name: coder warmers
+  description: The socks of the arms. Keeps things nice and cozy.
+  components:
+  - type: Sprite
+    sprite: Clothing/Hands/Gloves/coderwarmer.rsi
+    layers:
+    - state: icon
+  - type: Clothing
+    sprite: Clothing/Hands/Gloves/coderwarmer.rsi
+    clothingVisuals:
+      gloves:
+      - state: equipped-HAND
+  - type: Fiber
+    fiberColor: fibers-pink
+
+- type: entity
+  parent: ClothingHandsArmWarmers
+  id: ClothingHandsArmWarmersValid
```

Comment:
no

Reply:
no to the item in general or just id:valid